src/util/time.h

```diff
@@ -27,6 +27,12 @@ void UninterruptibleSleep(const std::chrono::microseconds& n);
 inline int64_t count_seconds(std::chrono::seconds t) { return t.count(); }
 inline int64_t count_microseconds(std::chrono::microseconds t) { return t.count(); }
 
+/**
+ * Helper to count the seconds in any std::chrono::duration type
+ */
+using SecondsDouble = std::chrono::duration<double, std::ratio<1, 1>>;
+inline double CountSeconds(SecondsDouble t) { return t.count(); }
```

Comment:
To my understanding, new code should follow the naming convention:

> Class names, function names, and method names are UpperCamelCase

instead of trying to mimic existent code.

Reply:
That was my reasoning as well.

Reply:
Personally I think this is an example of where the naming convention is buggy rather than the code; but especially if `int64_t count_seconds(micros)` should be `int64_t CountSeconds(seconds)` we shouldn't be introducing `double CountSeconds(SecondsDouble)` with a name clash like that -- would `CountSeconds(micros)` return an int or double?

Reply:
I'd personally just prefer `std::chrono::duration<double>(t).count()` instead of a helper.

Reply:
Renamed to `CountSecondsDouble`